spec/lib/tasks/data_import_spec.rb

```diff
@@ -29,13 +32,22 @@
         allow(Imports::OrganisationImportService).to receive(:new).and_return(import_service)
       end
 
-      it "creates an organisation from the given XML file" do
+      it "creates an organisation from the given XML file when the VCAP_SERVICES environment variable exists" do
         expect(Storage::S3Service).to receive(:new).with(paas_config_service, instance_name)
         expect(Imports::OrganisationImportService).to receive(:new).with(storage_service)
         expect(import_service).to receive(:create_organisations).with(fixture_path)
 
         task.invoke(type, fixture_path)
       end
+
```

Comment:
I didn't think it was worth adding a PaaS/Env test (like this one) for every context block in this file. I felt that testing it once was sufficient.

Reply:
You're probably right, I can't see another context that might interfere but if you can imagine another context that might affect this test that I would have it in 2, not all of them but just more than 1.

Reply:
Thanks. We can't think of another context right now so are happy with just 1 test, but also we're going to change how this works later on when we stop using PaaS, so think time spent on this now might end up being wasted anyway.